pkg/flag/misconf_flags.go

```diff
@@ -34,7 +34,7 @@ var (
 	IncludeNonFailuresFlag = Flag[bool]{
 		Name:       "include-non-failures",
 		ConfigName: "misconfiguration.include-non-failures",
-		Usage:      "include successes and exceptions, available with '--scanners misconfig'",
+		Usage:      "include successes and failures, available with '--scanners misconfig'",
```

Comment:
Failures are enabled by default, this flag additionally enables successes

Reply:
fixed